fileserver/routes.py

```diff
@@ -317,6 +317,14 @@ def get_session_version():
         return error_resp(http.NOT_FOUND)
     project = "session-foundation/session-" + platform
 
+    platform_os = request.args.get("os")
+    platform_arch = request.args.get("arch")
+    release_channel = request.args.get("release_channel") if request.args.get("release_channel") else 'latest'
+
+    # Backwards compatibility from before apple silicon builds
+    if platform_os is not None and platform_os == 'darwin' and platform_arch is None:
+        platform_arch = 'x86_64'
+
```

Comment:
I think that the change to include these values in requests + db storage warrants some more discussion (perhaps in a separate PR) around being able to retrieve the alpha version.

The more fields we have, the more possibility that some combination of those fields could pinpoint a specific user's activity.  (For example: if we added arm64 linux builds at some point, and there is only one linux arm64 user on the alpha release channel then that information would effectively allow you to tease out when that particular user is online).

Reply:
Turns out we don't need it at all for apple silicon builds. So unless we want that info probably don't need that second PR?